src/pages/Login/LoginFormContainer.test.jsx

```diff
@@ -45,7 +45,7 @@ describe('LoginFormContainer', () => {
       });
 
       expect(dispatch).toBeCalledWith({
-        type: 'changeLoginField',
+        type: 'application/changeLoginField',
```

Comment:
application 상수화해서 관리하셔도 괜찮을 것 같아요! 이름이 길어서 오타 같은 것도 방지할 수 있고!

Reply:
```suggestion
const APP = application;

        type: `${APP}/changeLoginField`,
```

이런 방법으로 `const APP`을 같은 파일에 선언하여 상수화 하는게 맞을까요..?

Reply:
- 이렇게 문자열로 표현해주셔야 할 것 같아요! 
```js
const APP = "application";
```